index.html

```diff
@@ -2,334 +2,298 @@
-    <script>
-        window.dataLayer = window.dataLayer || [];
-        function gtag() { dataLayer.push(arguments); }
-        gtag('js', new Date());
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>My Portfolio</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-81KZDKHG1W"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
 
-        gtag('config', 'G-81KZDKHG1W');
-    </script>
+    gtag("config", "G-81KZDKHG1W");
+  </script>
 </head>
 
 <body>
-    <header>
-        <div class="container">
-            <nav class="nav-item1">My Logo
-            </nav>
-            <nav class="nav-item3">
-                <img src="images/toggle.png" alt="toggle">
-            </nav>
-            <nav class="web-nav">
-                <ul class="web-nav-ul">
-                    <li class="web-nav-list">
-                        <a href="#Portfolio" class="web-nav-link">Portfolio</a>
-                    </li>
-                    <li class="web-nav-list">
-                        <a href="#About" class="web-nav-link">About</a>
-                    </li>
-                    <li class="web-nav-list">
-                        <a href="#Contact" class="web-nav-link">Contact</a>
-                    </li>
-                </ul>
-            </nav>
-        </div>
-    </header>
-    <section class="content">
-        <h1 class="box1">I’m Emmanuel
-            Glad to see you!</h1>
-        <p class="box2">
-            I'm a software developer! I can help you build a product,
-            feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded, don't hestiate to contact me.
-        </p>
-        <h3 class="box3">
-            LET'S CONNECT
-        </h3>
-        <ul class="box4">
-            <li class="social-media">
-                <img src="images/twitter.png" alt="twitter">
-            </li>
-            <li class="social-media">
-                <img src="images/Linkedin.png" alt="Linkedln">
-            </li>
-            <li class="social-media">
-                <img src="images/m.png" alt="m">
-            </li>
-            <li class="social-media">
-                <img src="images/github.png" alt="github">
-            </li>
-            <li class="social-media">
-                <img src="images/fingers.png" alt="two fingers">
-            </li>
+  <header>
+    <div class="container">
+      <nav class="nav-item1">My Logo</nav>
+      <div class="nav-item3">
+        <span class="hamburger-span" onclick="openNav()"> &#9776; </span>
+      </div>
```

Comment:
- [ ] Nice job 👏  👏  On implementing the mobile menu functionality I can see it is working properly, however one of the requirements says that you should use Javascript best practice. I think your project will be well structured if you put all Javascript functionality(events) in your `main.js file`. Kindly remove this `onclick` event here in your HTML and use `addEventListener` in `main.js`
#### An example in your main.js
```js
    document.querySelector('hamburger-span').addEventListener('click', openNav)
```